### DISCUSSION TITLE
Welcome to MOV.AI Flow™ Discussions!

### DISCUSSION THREAD

**Discussion by xsoulp** [MOV-AI/movai-flow#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xsoulp** [comment]
my first comment

**Reply by xsoulp** [reply]
i am replying to my own comment

**Reply by AlexFernandes-MOVAI** [reply]
Hello
